lib/launchApplication.js

```diff
@@ -29,12 +32,29 @@ module.exports = async function launchApplication(isDebug) {
         });
     }
 
-    let results = await controller.start();
+    if (wgtFilePath) {
+        controller.addStep({
+            title: 'Package ID',
+            placeholder: 'Enter package id',
+            validator: InputValidator.checkPackageId
+        });
+    }
 
-    let webApp = TVWebApp.openProject(vscode.workspace.rootPath);
-    if (webApp == null) {
-        logger.log('launchApplication:web App is null');
-        return;
+    let results = await controller.start();
+    
+    let webApp;
+    if (wgtFilePath) {
+        if (!fs.existsSync(wgtFilePath)) {
```

Comment:
I think it would be better to use `async/await` instead of a sync call

Reply:
Yes, I think I saw some references to `fs.readFileSync` inside repo hence took this approach to maintain consistency.